packages/zoe/src/contracts/autoswap.js

```diff
@@ -276,7 +236,50 @@ export const makeContract = harden(zcf => {
           },
           getLiquidityIssuer: () => liquidityIssuer,
           getPoolAllocation,
-          makeInvite,
+
+          // TODO review comment to be removed
```

Comment:
Ok, I think I have an idea that can simplify this, but I will try in a PR after this is merged to master.

Reply:
Looking forward!